integ-test/src/test/resources/correctness/expressions/date_and_time_functions.txt

```diff
@@ -0,0 +1,13 @@
+DATE '2001-05-07'
+TIME '12:30:00'
```

Comment:
what about nanoseconds, does that work? 
```
TIME '12:30:00.004'
```

Reply:
Interestingly `TIME '12:30:00.004` does not work but `TIMESTAMP '2001-05-07 12:30:00.1234'` does work

Reply:
After investigating I have found that this does not work because of a known bug https://github.com/opensearch-project/sql-jdbc/issues/16